plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/messages.properties

```diff
@@ -543,6 +543,8 @@ VCO_REGEX_ILLEGAL_ESCAP=error;;;Illegal escape sequence in regular expression {0
 VCO_NPE=error;;;A NullPointerException occurred. This indicates a missing value converter or a bug in its implementation.
 # 0: feature name
 VCO_NULL_FEATURE=error;;;ValueConverter returned null for primitive feature {0}
+# 0: variable name
+VCO_USED_BEFORE_DECLARED=warning;;;Variable {0} is used before it is declared
```

Comment:
Why prefix "VCO_"? Is it for value converter I think

Reply:
I'll change that